client/__tests__/e2e/playwright.global.setup.ts

```diff
@@ -42,14 +42,13 @@ const setup = async ({ page }: { page: Page }) => {
       },
     });
   });
-
   page.on("console", (message) => {
     if (message.type() === "error") {
-      throw new Error(`CLIENT SIDE ERROR: ${  message.text()}`);
+      throw new Error(`CLIENT SIDE ERROR: ${message.text()}`);
```

Comment:
I think we shoud do `JSON.stringify` on the message, so we get more context, such as the API URL in addition to the text. What do you think?
```suggestion
      throw new Error(`CLIENT SIDE ERROR: ${JSON.stringify(message)}`);
```